src/components-examples/cdk/stepper/cdk-linear-stepper-with-form/cdk-linear-stepper-with-form-example.ts

```diff
@@ -0,0 +1,43 @@
+import {Component, OnInit} from '@angular/core';
+import {CdkStepper} from '@angular/cdk/stepper';
+import {FormBuilder, FormGroup, Validators, FormsModule, ReactiveFormsModule} from '@angular/forms';
+
+/** @title A custom CDK linear stepper with forms */
+@Component({
+  selector: 'cdk-linear-stepper-with-form',
+  templateUrl: './cdk-linear-stepper-with-form-example.html',
+  styleUrls: ['./cdk-linear-stepper-with-form-example.css']
+})
+export class CdkLinearStepperWithFormExample {
+  isLinear = true;
+  firstFormGroup: FormGroup;
+  secondFormGroup: FormGroup;
+
+  constructor(private _formBuilder: FormBuilder) {}
+
+  ngOnInit() {
+    this.firstFormGroup = this._formBuilder.group({
```

Comment:
I'm not familiar with what's generally recommended in this case, but I'm wondering if it would be better to do this in the constructor or inline where the fields are declared. The advantage of doing either of those is that firstFormGroup and secondFormGroup could be marked as readonly.

Reply:
AFAIK this could go in the constructor (I'm about 80% sure of that myself)

Reply:
Yes I was able to move it to the constructor and make it readonly without issue.